src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxNode.cs

```diff
@@ -112,6 +112,8 @@ private static SyntaxTree ComputeSyntaxTree(CSharpSyntaxNode node)
             return tree;
         }
 
+        public abstract TResult? Accept<TArgument, TResult>(CSharpSyntaxVisitor<TArgument, TResult> visitor, TArgument argument);
```

Comment:
❗ Incorrect return value here

```suggestion
        public abstract TResult Accept<TArgument, TResult>(CSharpSyntaxVisitor<TArgument, TResult> visitor, TArgument argument);
```